### ISSUE TITLE
"Browse" returns no rows, even if there are rows

### ISSUE THREAD

**Issue by tingvold** [HuasoFoundries/phpPgAdmin6#333]
PostgreSQL 9.6.15
phpPgAdmin6 on latest master-branch (16b6650726173410dd30ef1274da96d64d83d368)
nginx/1.18.0
php7.3 (7.3.19-1~deb10u1)

Trying to hit "Browse" on a table returns "No rows found";

![image](https://user-images.githubusercontent.com/359918/92598089-9c5d2900-f2a8-11ea-99ee-b52e46c65e1d.png)

Using the same query in the "SQL" (sqledit, top right corner), results in correct rows being returned;

![image](https://user-images.githubusercontent.com/359918/92598146-b4cd4380-f2a8-11ea-8c8a-5ca4e33410b3.png)

**Comment by tingvold** [comment]
Video showing the issue (including issue #333 and #334): https://files.jocke.no/debug/phppgadmin6-issues_20200909.mov

**Comment by tingvold** [comment]
As a (moot) comparison, this works fine using phpPgAdmin v6.0.0-b28 on PostgreSQL 9.6.10.

**Comment by dcramble** [comment]
I can add that this also works in phpPgAdmin6 v6.0.0-beta.47 (while broken in the release version for me).

**Comment by ffflabs** [comment]
Can you check if it does work for you in release v6.1.x?

**Comment by dcramble** [comment]
I having trouble getting 6.1.2 to run at the moment. I'm getting:

 _The requested URL /src/views/login was not found on this server._  

when I try to login. When I copy back a different version, it works immediately.  By "copy" I'm just moving my different directories for v6-beta47, v6-release, and v6.1.2-tag into the path name for my apache alias directory.

I can at least confirm that when I get logged in to 6-release and get the rows issue, I can copy 6.1.2 into position on my web server, and the rows will display now - indicating the issue may be resolved. (It seems my issue with 6.1.2 is just getting through the sever login.)

**Comment by ffflabs** [comment]
weird... are you running it inside a subfolder?

**Comment by dcramble** [comment]
Yes, I am running it within a sub-folder. With the sub-folder set in the config.

**Comment by ffflabs** [comment]
is your subfolder config defined with a leading slash ?
I'm  pretty lost anyway. I've been trying to implement an url helper to output absolute urls for everything, but sometimes the subfolder is lost (therefore, I can't trust that relative urls will figure themselves out), others it's repeated (therefore, I can't just prepend the subfolder to every path).

**Comment by dcramble** [comment]
Yeah it does have a leading slash. I tried with and without a trailing slash as well.

For now - to stay on task with this specific issue, I moved the project to the web root and tried again. In the 6.1.3 version, this rows issues is fixed!
